static/styles/subscriptions.css

```diff
@@ -1001,40 +909,142 @@ h4.stream_setting_subsection_title {
+                            max-width: 20ch;
+                            overflow: hidden;
+                            text-overflow: ellipsis;
+                            float: left;
+                        }
+                    }
+
+                    .button-group {
+                        float: right;
+
+                        .deactivate {
+                            margin-right: 3px;
+                            text-decoration: none;
+                        }
+                    }
+
+                    .subscribe-button.unsubscribed {
+                        color: hsl(156, 39%, 54%);
+                        border-color: hsl(156, 39%, 77%);
+
+                        &:active {
+                            background-color: hsla(156, 39%, 54%, 0.2);
+                        }
+                    }
+
+                    .large-icon {
+                        display: inline-block;
+                        vertical-align: top;
+                        margin-right: 5px;
+                        margin-top: -5px;
+                        font-size: 1.5em;
+
+                        &.hash::after {
+                            top: 1px;
+                            font-size: 1.09em;
+                            font-weight: 800;
+                        }
+                    }
+
+                    .stream_change_property_info {
+                        vertical-align: top;
+                        margin: -5px 0 0;
+                    }
+                }
+
+                .stream_setting_subsection_header {
+                    display: flex;
+
+                    .stream_permission_change_info {
+                        /* TODO: Deduplicate with .save-discard-widget-button. */
+                        margin: 12px auto 12px 3px;
+                    }
+
+                    .button-group {
+                        display: inline;
+                        margin-left: auto;
+                        align-self: center;
+                    }
+                }
+
+                .subscription-type {
+                    margin-bottom: 10px;
+
+                    .subscription-type-text {
+                        display: inline;
+                    }
+
+                    b {
+                        font-weight: 600;
+                    }
+                }
+
+                .stream-description {
+                    margin-bottom: 5px;
+
+                    .no-description {
+                        font-style: italic;
+                        color: hsl(0, 0%, 67%);
+                    }
+                }
+
+                .stream_change_property_status {
+                    margin: 9px auto 3px 3px;
+                }
+            }
         }
     }
 
     .loading_indicator_text {
         font-weight: 400;
         line-height: 20px;
     }
-}
 
-#subscription_overlay .subsection-parent {
-    .input-group {
-        input[type="checkbox"] {
-            margin-top: 0;
-        }
```

Comment:
Since there already exists a CSS classname for `.subsection-parent` separately; i added the CSS of `input-group` inside it to avoid repeated classnames.